content/en/docs/examples/microservices-istio/enable-istio-all-microservices/index.md

```diff
@@ -0,0 +1,100 @@
+functionality for more microservices. For the purpose of this tutorial, you will
+enable Istio on all the remaining microservices in one step.
+
+1.  For the purpose of this tutorial, scale the deployments of the microservices
+    down to 1:
+
+    {{< text bash >}}
+    $ kubectl scale deployments --all --replicas 1
+    {{< /text >}}
+
+1.  Redeploy the Bookinfo application, Istio-enabled. `productpage` will not be
+    redeployed since it already has Istio injected, and its pods will not be
+    changed. This time you will use only a single replica of a microservice,
+    scaling of the microservices is orthogonal to the demonstrated Istio
+    features.
+
+    {{< text bash >}}
+    $ curl -s {{< github_file >}}/samples/bookinfo/platform/kube/bookinfo.yaml | istioctl kube-inject -f - | kubectl apply -l app!=reviews -f -
+    $ curl -s {{< github_file >}}/samples/bookinfo/platform/kube/bookinfo.yaml | istioctl kube-inject -f - | kubectl apply -l app=reviews,version=v2 -f -
+    service "details" unchanged
+    deployment "details-v1" configured
+    service "ratings" unchanged
+    deployment "ratings-v1" configured
+    service "productpage" unchanged
+    deployment "productpage-v1" configured
+    deployment "reviews-v2" configured
+    {{< /text >}}
+
+1.  Access the application's webpage several times. Note that Istio was added
+    **transparently**, the original application did not change. It was added on
+    the fly, without the need to undeploy and redeploy the whole application,
+    without hurting the application's availability.
+
+1.  Check the application pods and verify that now each pod has two containers.
+    One container is the microservice itself, the other is the sidecar proxy
+    attached to it:
+
+    {{< text bash >}}
+    $ kubectl get pods
+    details-v1-58c68b9ff-kz9lf        2/2       Running   0          2m
+    productpage-v1-59b4f9f8d5-d4prx   2/2       Running   0          2m
+    ratings-v1-b7b7fbbc9-sggxf        2/2       Running   0          2m
+    reviews-v2-dfbcf859c-27dvk        2/2       Running   0          2m
+    sleep-88ddbcfdd-cc85s             1/1       Running   0          7h
+    {{< /text >}}
+
+1.  Access the Istio dashboard at
+    [http://my-istio-dashboard.io/dashboard/db/istio-mesh-dashboard](http://my-istio-dashboard.io/dashboard/db/istio-mesh-dashboard).
+    In the top left drop-down menu, select _Istio Mesh Dashboard_. Note that now all the services from your namespace
+    appear in the list of services.
+
+    {{< image width="80%"
+        link="dashboard-mesh-all.png"
+        caption="Istio Mesh Dashboard"
+        >}}
+
+1.  Check some other microservice in _Istio Service Dashboard_, e.g. `ratings` :
+
+    {{< image width="80%"
+        link="dashboard-ratings.png"
+        caption="Istio Service Dashboard"
+        >}}
+
+1.  Visualize your application's topology by using the [Kiali](https://www.kiali.io) console. Access
```

Comment:
I suggest clarify that kiali is not a part of istio

Reply:
fixed